drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -133,6 +134,12 @@
                     <div id="item-text-color-description" class="item-styles-form-help">
                       {{ help_texts.item_text_color }}
                     </div>
+                    <label class="h3" title="{{ help_texts.max_items_per_zone }}">
+                        {% trans "Maximum items per zone" %}
+                        <br/>
+                        <input name="max-items-per-zone" id="max-items-per-zone" type="number" min="1" step="1" value="{{ self.max_items_per_zone|unlocalize }}" />
+                    </label>
+                    <span class="sr">{{ help_texts.max_items_per_zone }}</span>
```

Comment:
Don't think you need to duplicate the title text in a screen reader span (which isn't linked to the input anyway).